tests/test_functional.py

```diff
@@ -94,16 +94,6 @@ class Test8BitBlockwiseQuantizeFunctional:
     @pytest.mark.parametrize("blocksize", [4096, 2048, 1024, 512, 256, 128, 64])
     @pytest.mark.parametrize("signed", TRUE_FALSE, ids=id_formatter("signed"))
     def test_dynamic_blockwise_quantization(self, device, dtype, nested, blocksize, signed):
-        if device == "cpu":
-            # This test is slow on CPU, so avoid atypical use cases.
-            if nested:
-                pytest.skip("Not a typical use case.")
-            if blocksize != 256:
-                pytest.skip("Only blocksize 256 is the typical one supported on CPU.")
```

Comment:
I think we should still skip these, at least for the fp32 kernel. If the torch implementation for bf16/fp16 is that much faster then it may make sense to keep or put a `slow` marker on it so we could filter it out.

Reply:
done

Reply:
I still want to keep the bf16/fp16 tests because we may need to enable absmax with fp16/bf16 dtype in the future.